README.md

```diff
@@ -29,6 +29,7 @@ npm install vue-compose-promise
 <script>
 import { createComponent } from '@vue/composition-api'
 import { usePromise } from 'vue-compose-promise'
+// import { fetchUsers } from './data';
```

Comment:
I meant, completely removing the import statement and writing a comment above the first call of fetchUsers